CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # Changelog
 
+## 1.4 Beta 2 
+
+### Additions and Improvements 
+
+- Enable TLS for JSON-RPC HTTP Service [\#253](https://github.com/hyperledger/besu/pull/253)
+
+Exposes =command line parameters to enable TLS on Ethereum JSON-RPC HTTP interface to allow clients like EthSigner to connect via TLS: 
```

Comment:
Should that say "Expose following new command line parameters"

Reply:
Fixed